resources.py

```diff
@@ -0,0 +1,135 @@
+import datetime
```

Comment:
same decorators comment for this file too; marking as viewed for now, will review again if you implement decorators